UVtools.Core/Operations/OperationPixelArithmetic.cs

```diff
@@ -646,6 +680,22 @@ protected override bool ExecuteInternally(OperationProgress progress)
                         case PixelArithmeticOperators.DiscardRegion:
                             target.SetTo(EmguExtensions.BlackColor);
                             break;
+                        case PixelArithmeticOperators.Corrode:
+                            {
+                                var pixels = target.GetDataByteSpan();
+
+                                for (int i = pixels.Length - 1; i >= 0; i--)
+                                {
+                                    if (!_preserveVoids || pixels[i] > 0)
```

Comment:
What `_preserveVoids` is for? It seen it must be false for this method to work?

Reply:
Preserve voids keeps black pixels black, the logic here ensures that all pixels are processed when false, or when true that the input pixel is black. This is the same as the model mask, but eliminating the random call for black pixels when not wanted results in a significant speedup.